docs/diagrams/bc-registries-audit-decision-tree.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
+
+:**Error Messages from BC Registries Audit Notification Received;
+start
+:Confirm the alert by going to the 
+""Orgbook.gov.bc.ca"" url;
```

Comment:
Provide additional details on how to verify the issue.

Reply:
Additional deteailes provided: Confirm the alert by browsing to ""Orgbook.gov.bc.ca"" to verify 
the alert received in BC Registries Audit Notification is not false positive 
by ensuring the data of Orgbook matches with BC Registries;